src/BuildScriptGeneratorCli/Commands/CommandBase.cs

```diff
@@ -43,6 +45,30 @@ public int OnExecute(CommandLineApplication app, IConsole console)
                 if (envSettings != null && envSettings.GitHubActions)
                 {
                     logger?.LogInformation("The current Oryx command is being run from within a GitHub Action.");
+
+                    // Format: "2020-02-15T02:51:50.000Z"
+                    var gitHubActionBuildContainerStartTime = envSettings.GitHubActionsBuildContainerStartTime;
+                    var gitHubActionBuildContainerEndTime = envSettings.GitHubActionsBuildContainerEndTime;
+
+                    var dateStart = gitHubActionBuildContainerStartTime.Split('T')[0];
+                    var timeStart = gitHubActionBuildContainerStartTime.Split('T')[1];
+
+                    var dateEnd = gitHubActionBuildContainerEndTime.Split('T')[0];
+                    var timeEnd = gitHubActionBuildContainerEndTime.Split('T')[1];
+
+                    DateTime date1 = new DateTime(int.Parse(dateStart.Substring(0, 4)), int.Parse(dateStart.Substring(5, 2)), int.Parse(dateStart.Substring(8, 2)),
+                                                  int.Parse(timeStart.Substring(0, 2)), int.Parse(timeStart.Substring(3, 2)), int.Parse(timeStart.Substring(6, 2)));
+                    DateTime date2 = new DateTime(int.Parse(dateEnd.Substring(0, 4)), int.Parse(dateEnd.Substring(5, 2)), int.Parse(dateEnd.Substring(8, 2)),
+                                                  int.Parse(timeEnd.Substring(0, 2)), int.Parse(timeEnd.Substring(3, 2)), int.Parse(timeEnd.Substring(6, 2)));
+                    TimeSpan interval = date2 - date1;
+                    var gitHubActionBuildContainerDurationSeconds = interval.Seconds.ToString();
+                    var buildEventProps = new Dictionary<string, string>()
+                    {
+                        { "gitHubActionBuildContainerStartTime", gitHubActionBuildContainerStartTime },
+                        { "gitHubActionBuildContainerEndTime", gitHubActionBuildContainerEndTime },
+                        { "gitHubActionBuildContainerDurationSeconds", gitHubActionBuildContainerDurationSeconds },
```

Comment:
@qianz2 nit: if we're sending the elapsed time for building the action, I don't think we need to also send the start and end times (I can't think of any way that we'd use those pieces of information on top of the elapsed time)